presto-hive/src/test/resources/hive_symlink_table/_symlink_format_manifest/manifest

```diff
@@ -0,0 +1 @@
+file:/Users/reetika/Documents/workspace/personal/prestodb/presto-hive/target/test-classes/hive_symlink_table/part-00000-88a5c9b8-81e2-44dc-b696-beac42f65d69-c000.snappy.parquet
```

Comment:
Should we update this manifest file? Would this location work?

Reply:
Missed this. Udated